check-tcp/check-tcp_test.go

```diff
@@ -235,7 +235,7 @@ func TestUnixDomainSocket(t *testing.T) {
 	testUnexpected()
 
 	testOverWarn := func() {
-		opts, err := parseArgs([]string{"-U", sock, "--send", `PING`, "-E", "-e", "OKOK", "-w", "0.1"})
+		opts, err := parseArgs([]string{"-U", sock, "--send", `PING`, "-E", "-e", "OKOK", "-w", "0.000000001"})
```

Comment:
[suggestion] define opts.Critical as time.Duration. you can specify this value like `-w 3s`.